express-api/src/controllers/administrativeAreas/administrativeAreasController.ts

```diff
@@ -20,9 +24,14 @@ export const getAdministrativeAreas = async (req: Request, res: Response, next:
       }]
    */
   try {
+    const ssoUser = req.user as SSOUser;
     const filter = AdministrativeAreaFilterSchema.safeParse(req.query);
     if (filter.success) {
       const adminAreas = await administrativeAreasServices.getAdministrativeAreas(filter.data);
+      if (!ssoUser.client_roles || !ssoUser.client_roles.includes(Roles.ADMIN)) {
```

Comment:
Its preferred to use the `hasRoles` function. https://github.com/bcgov/citz-imb-sso-express/wiki/06-Getting-User-Information